pkg/systemd/terminal.jsx

```diff
@@ -70,29 +89,29 @@
             this.setState({ title });
         }
 
-        setCookie(key, value) {
-            const cookie = key + "=" + encodeURIComponent(value) +
-                         "; path=/; expires=Sun, 16 Jul 3567 06:23:41 GMT";
+        invalidateCookie(key) {
+            const cookie = key + "=''" +
+                         "; path=/; Max-Age=0;";
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18499-20230314-183936-d376ccc5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)